src/browser/commands/types.ts

```diff
@@ -64,6 +64,21 @@ export interface AssertViewOpts extends Partial<AssertViewOptsConfig> {
      * @defaultValue `true`
      */
     disableAnimation?: boolean;
+    /**
+     * Ability to ignore small amount of different pixels while screenshots are still being considered as "same"
+     *
+     * @example 5
+     * @example '1.5%'
+     *
+     * @remarks
+     * Usefull, when you have 2-5 different pixels you can't get rid of using `tolerance` and `antialiasingTolerance`.
+     *
+     * @note
+     * This should be considered a last resort and only used in small number of cases where necessary.
+     *
+     * @defaultValue `0`
+     */
+    maxDiffPixels?: `${number}%` | number;
```

Comment:
Turns out, you can do such things in typescript.
Using `` `${number}%` | number `` directly, without extra type, because this way you can get clear understanding about what type this field wants value to be:
![image](https://github.com/gemini-testing/hermione/assets/54893992/43549ea7-0578-4b86-b5c2-fd28dde598e2)